packages/enzyme-adapter-react-16/package.json

```diff
@@ -40,12 +40,12 @@
     "object.values": "^1.0.4",
     "prop-types": "^15.6.0",
     "react-reconciler": "^0.7.0",
-    "react-test-renderer": "^16.0.0-0"
+    "react-test-renderer": "^16.0.0-0  || ^16.3.0-0"
```

Comment:
Why do we need to do this?

Reply:
`^16.0.0-0` only handles prerelease ranges the exact version `16.0.0` not any prerelease of any version of `16`